events/event_schema.json

```diff
@@ -57,5 +57,6 @@
       }
     }
   },
-  "required": ["title", "location", "url"]
+  "required": ["title", "location", "url"],
+  "additionalProperties": false
```

Comment:
`additionalProperties: false` needs to show up on every level of the schema as implemented already on [master](https://github.com/coderetreat/coderetreat.github.io/blob/master/events/event_schema.json). I think we can stick to the version on `master`, but I'd love to have you submit another PR for the test below.